runtime/src/genesis_utils.rs

```diff
@@ -10,11 +11,15 @@ use solana_sdk::{
     system_program,
 };
 use solana_stake_program::stake_state;
+use solana_stake_program::stake_state::StakeState;
 use solana_vote_program::vote_state;
 use std::borrow::Borrow;
 
 // The default stake placed with the bootstrap validator
-pub const BOOTSTRAP_VALIDATOR_LAMPORTS: u64 = 42;
```

Comment:
When checking the overall effect of this overage rewrite, I noticed that many tests are actually creating really small stake accounts under the rent-exempt threshold. I wanted to get rid of them because tests doesn't reflect reality and it's confusing. Also, to get good exposure, I wanted overage check always-on when testing.

After some dirties hacks, it turns out this is doable. (now ci passes).

Reply:
> To confirm: only (3) really matters, but you're changing all of them for because it's easier, right?

Yeah, I didn't dig in at all. I just wanted to pass the CI to know my rewrite's full consequences using our test assets.



> Could you pull that change out into a separate PR? It's quite a bit of churn on this one.

Yeah, I'll create a separate pr.

Reply:
or your recent #13627 improves the situation? :) I'll take a look at it, too.